mpdev/cmd/applycmd.go

```diff
@@ -0,0 +1,111 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/GoogleCloudPlatform/marketplace-tools/mpdev/internal/apply"
+	"github.com/GoogleCloudPlatform/marketplace-tools/mpdev/internal/docs"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v3"
+)
+
+func GetApplyCommand() *cobra.Command {
+	var c Command
+	cmd := &cobra.Command{
+		Use:     "apply",
+		Short:   docs.ApplyShort,
+		Long:    docs.ApplyLong,
+		Example: docs.ApplyExamples,
+		RunE:    c.RunE,
+	}
+
+	cmd.Flags().StringSliceVarP(&c.Filenames, "filename", "f", c.Filenames, "that contains the configuration to apply")
```

Comment:
The help output currently reads:

```
Flags:
  -f, --filename strings   that contains the configuration to apply
  -h, --help               help for apply
```

Perhaps we should reword to `file(s) from which configurations are read from`.

Reply:
I copied this wording directly from `kubectl apply` https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#apply